app/default/routes.py

```diff
@@ -249,12 +249,25 @@ def submit_application():
         json=payload,
     )
     submitted = submission_response.json()
+    application_id = submitted.get("id")
+    application_reference = submitted.get("reference")
+    application_email = submitted.get("email")
+    if not application_reference or not application_email:
+        raise Exception(
+            "Unexpected response from application store when submitting application"
+            " application: "
+            + str(application_id) +
+            "application-store-response: "
+            + str(submission_response)
```

Comment:
Can we check the response code from the POST further up? e.g. like we do e.g. in https://github.com/communitiesuk/funding-service-design-frontend/blob/880ebe5e55f8bb5c07076477db71b383a8684f0e/app/default/routes.py#L127 rather than parsing the response body?

Reply:
Also, we could do hard lookup and fail earlier rather than `.get()` which may return `None` and cause us problems later on?

Reply:
No probs, updated